streamer_test.go

```diff
@@ -233,3 +235,20 @@ func TestStreamerFlushCount(t *testing.T) {
 	streamer.Close()
 	client.AssertFlushCount(t, 2)
 }
+
+func TestNewStreamerMutuallyExclusiveConfigErr(t *testing.T) {
+	storageClient := new(StorageClientConfig)
+	storageClient.BigQuerySchema = &bq.Schema{}
```

Comment:
Not a biggie, but I would prefer you choose a syntax, either `new` or `${}` :P especially combining them next to one another screams at me somehow

Reply:
I've changed the usages of `${}` for `new(...)` as that seems the cleaner syntax to use.